server/src/server/server.ts

```diff
@@ -8,7 +8,7 @@ import * as rateLimit from 'express-rate-limit';
 export function createServer(app: App) {
   const server = express();
   server.use(helmet());
-  server.use(rateLimit({ windowMs: 1 * 60 * 1000, max: 20 })); // Limit each IP to 20 requests per minute
+  server.use(rateLimit({ windowMs: 1 * 60 * 1000, max: 35 })); // Limit each IP to 35 requests per minute
```

Comment:
I'd say let's set it to 1000, then we can revisit it later before the final release. @BambanzaJuniorThe2nd 
Of course we'll also need to optimize our requests at some point.